### PR TITLE
Fix `close-out-of-view-modals` on more pages

### PR DESCRIPTION
**Pull request by yakov116**
Did some more debugging.  This should be correct.

Also fixed the error when setting your status
<!--

Thanks for contributing! 🍄

1. LINKED ISSUES:
   Does this PR close/fix an existing issue? Write something like `Closes #10`

2. TEST URLS:
   Add some test URLs

3. SCREENSHOT:
   Add a screenshot here if your PR makes visual changes

### CODE REVIEW

**Comment by fregante** [comment]
I'm not confident that `details-menu` is used everywhere, especially in older `.dropdown-menu` items

List where you tested this or else we're doing whack-a-mole.
Perhaps include each tested dropdown in the code, e.g.:

```js
// This should work on:
// "Labels" dropdown in issues
// "Watch" dropdown on repos’ headers
```
and so on.

File: source/features/close-out-of-view-modals.tsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
 
 function init(): void {
 	// The `open` attribute is added after this handler is run, so the selector is inverted
-	delegate('.details-overlay:not([open]) > summary', 'click', ({delegateTarget: summary}) => {
+	delegate('.details-overlay:not([open]) > summary[aria-haspopup="menu"]', 'click', ({delegateTarget: summary}) => {
```
Comment (fregante) [inline source/features/close-out-of-view-modals.tsx]: I'd avoid this extra check, there's only ever one `summary`

Comment (yakov116) [reply source/features/close-out-of-view-modals.tsx]: Nope, Let me get you a few links

**Comment by yakov116** [comment]
> ```js
> // This should work on:
> // "Labels" dropdown in issues
> // "Watch" dropdown on repos’ headers
> ```


Side note: `"Labels" dropdown in issues,`

 With sticky side bar off....

File: source/features/close-out-of-view-modals.tsx (hunk 1 above)
Comment (fregante) [reply source/features/close-out-of-view-modals.tsx]: Indeed, hopefully GitHub settled on `[aria-haspopup="menu"]' + details-menu` for all. Maybe we should also make sure all of our dropdown match this as well.

**Comment by fregante** [comment]
>  Did github add stick-disscussopn-sidebar? i turned it off and its still sticky

I disabled Refined GitHub, zoomed out to 25% and the sidebar isn't sticky. Did you load RG twice?

**Comment by yakov116** [comment]
> > Did github add stick-disscussopn-sidebar? i turned it off and its still sticky
> 
> I disabled Refined GitHub, zoomed out to 25% and the sidebar isn't sticky. Did you load RG twice?

Ahh now I see it

File: source/features/close-out-of-view-modals.tsx (hunk 1 above)
Comment (yakov116) [reply source/features/close-out-of-view-modals.tsx]: In this PR?

Comment (yakov116) [reply source/features/close-out-of-view-modals.tsx]: @fregante Done

**Comment by fregante** [comment]
Excellent! This is the kind of solutions I like

**Pull request merged**